config/migrate.go

```diff
@@ -5,6 +5,7 @@ package config
 
 import "github.com/pkg/errors"
 
+// Migrate migrates SAML keys and certificates.
```

Comment:
Let's also mention from where and to where does it migrate.

Reply:
The function signature shows it pretty clearly:

```go
 func Migrate(from, to string) error {
```

The doc would sound a bit strange:
```go
 // Migrate migrates SAML keys and certificates from "from" to "to".
```

Alternatively, we could rename the parameter:

```go
 func Migrate(src, dst string) error {
```

so that we can say:

```go
 // Migrate migrates SAML keys and certificates from "src" to dst".
```

which one do you think looks better?

Reply:
Ah no, I meant what do from and to stand for ? If we just say "src" and "dst", it's not clear what they are. I was expecting something like "Migrate .. from a given <x> to another <x>". Now that I see the code, it can be a file path or a database DSN.

Perhaps something like: "Migrate migrates .. from one store to another given their data source names" ? Since "data source name" is the term used in "NewStore".

Reply:
Right, good idea, "data source name" is used in "NewStore". I've just pushed the update.